handbook/engineering/deployments.md

```diff
@@ -88,6 +88,62 @@ The other clusters are deployed and rolled back in the same way as sourcegraph.c
 	```
 	kubectl get pods --all-namespaces
 	```
+ 
+ ## How to start a test cluster in our "Sourcegraph Auxiliary' project on GCP
+ 
+- Go to [Sourcegraph Auxiliary](https://console.cloud.google.com/kubernetes/list?project=sourcegraph-server)
+- Click create a cluster.
+- Give it a name (a good convention is to prefix with your username).
+- Keep the defaults zonal and us-central1.
+- Set the default pool to 3 nodes.
+- Change machine type to n1-standard-8.
```

Comment:
Recommendation in [deploy-sourcegraph here]](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/k8s.md) is n1-standard-16 — does `n1-standard-8` also work?

Reply:
changed to -16

depending on what you're testing -8 works but it's simpler to give clear instructions